app/screens/VenueScreen/VenueScreen.tsx

```diff
@@ -1,14 +1,23 @@
 import React, { FC } from "react"
-import { SectionList, TextStyle, View, ViewStyle } from "react-native"
+import { SectionList, SectionListData, TextStyle, View, ViewStyle } from "react-native"
 import { Carousel, Screen, Text } from "../../components"
 import { TabScreenProps } from "../../navigators/TabNavigator"
 import { colors, spacing } from "../../theme"
 import { useHeader } from "../../hooks/useHeader"
 import { translate } from "../../i18n"
 import { SponsorCard } from "./SponsorCard"
+import { useSponsors, useVenues } from "../../services/api"
+import { WEBFLOW_MAP } from "../../services/api/webflow-consts"
+import { groupBy } from "../../services/api/webflow-helpers"
+import { RawSponsor } from "../../services/api/webflow-api.types"
 
-const irImage1 = require("../../../assets/images/info-ir1.png")
-const irImage2 = require("../../../assets/images/info-ir2.png")
+const sponsorTiers = Object.values(WEBFLOW_MAP.sponsorTier)
+type Tiers = typeof sponsorTiers[number]
+
+const initialTiers = sponsorTiers.reduce<Record<Tiers, RawSponsor[]>>(
+  (acc, tier) => ({ ...acc, [tier]: [] }),
+  {} as Record<Tiers, RawSponsor[]>,
+)
```

Comment:
This was a blocker for some time that I had to ask my 6'5" elf to help me with 😅

Reply:
LOL